docs/concepts/multi-region.md

```diff
@@ -1,4 +1,45 @@
-# KubeCarrier - Multi Region API
+# KubeCarrier API and multi region support
+
+## API
+
+KubeCarrier is meant as a central component in the IT stack of a company.
+
+To drive integration with other tools, KubeCarrier needs to expose an API that can be used to drive CLIs, Client-Side Operators, Service Catalog Adapters, UIs and more.
+
+### Goals
+
+- Allows external tools to interact with and extend KubeCarrier
+- Common interface for tooling
+
+### Requirements
+
+- MUST not require to expose the kube-apiserver on public networks
```

Comment:
What is the rationale behind this requirement? Will our possible API server be exposed on public networks?

Reply:
It seems to be common consensus to not expose the kube-apiserver on the public network, due to security concerns.
Hard to find something in K8s docs, but as reference:

> As the Kubernetes API server isn’t (and shouldn’t be) accessible from the public network, some workloads must use a separate system to authenticate. An example of this is authenticating across clusters, such as from within a cluster to somewhere else.

https://sysdig.com/blog/whats-new-kubernetes-1-18/
#1393 Provide OIDC discovery for service account token issuer

Reply:
Can we reuse superCluster kube-apiserver as our root entry point and assume the users shall enforce firewalling kube-apiserver behind VPN

Reply:
Maybe... although requiring external firewalls and restrictions will make setting up KubeCarrier very complicated